src/main/features/certificateOfService/views/document-upload.njk

```diff
@@ -0,0 +1,124 @@
+                      <a target="_blank" href="/legal/certificateOfService/document-download?selfPath={{ file.documentManagementURI }}&fileName={{ file.fileName }}">{{ file.fileName }}</a>
+                    </li>
+                  {% endfor %}
+                </ul>
+              </div>
+            {% endif %}
+            {% if fileToUpload.value === DocumentType.MEDICAL_REPORTS.value %}
+              {% include "includes/file-upload-form.njk" %}
+            {% else %}
+              <input type="submit" name="medicalReport" class="button button-secondary" value="{{ t('Add file') }}"{% if fileToUpload %} disabled{% endif %} />
+            {% endif %}
+          </div>
+        {% endif %}
+
+        {% if whatDocuments.types.indexOf(DocumentType.SCHEDULE_OF_LOSS.dataStoreValue) != -1 %}
+          <div class="form-group">
+            <label class="form-label-bold">{{ t('Schedule of loss') }}</label>
+            {% if scheduleOfLoss.length > 0 %}
+              <div class="form-group">
+                <ul class="list-number list-border">
+                  {% for file in scheduleOfLoss %}
+                    <li>
+                      <a target="_blank" href="/legal/certificateOfService/document-download?selfPath={{ file.documentManagementURI }}&fileName={{ file.fileName }}">{{ file.fileName }}</a>
+                    </li>
+                  {% endfor %}
+                </ul>
+              </div>
+            {% endif %}
+            {% if fileToUpload.value == DocumentType.SCHEDULE_OF_LOSS.value %}
+              {% include "includes/file-upload-form.njk" %}
+            {% else %}
+              <input type="submit" name="scheduleOfLoss" class="button button-secondary" value="{{ t('Add file') }}"{% if fileToUpload %} disabled{% endif %} />
+            {% endif %}
+          </div>
+        {% endif %}
+
+        {% if whatDocuments.types.indexOf(DocumentType.OTHER.dataStoreValue) != -1 %}
+          <div class="form-group">
+            <label for="other" class="form-label-bold">{{ t('Other') }}</label>
+            <label for="other" class="form-hint">{{ whatDocuments.otherDocuments }}</label>
+            {% if other.length > 0 %}
+              <div class="form-group">
+                <ul class="list-number list-border">
+                  {% for file in other %}
+                    <li>
+                      <a target="_blank" href="/legal/certificateOfService/document-download?selfPath={{ file.documentManagementURI }}&fileName={{ file.fileName }}">{{ file.fileName }}</a>
+                    </li>
+                  {% endfor %}
+                </ul>
+              </div>
+            {% endif %}
+            {% if fileToUpload.value === DocumentType.OTHER.value %}
+              {% include "includes/file-upload-form.njk" %}
+            {% else %}
+              <input type="submit" name="other" class="button button-secondary" value="{{ t('Add file') }}"{% if fileToUpload %} disabled{% endif %} />
```

Comment:
are we sure we want to be disabling buttons?
there's quite strong guidance from GDS we shouldn't be...
the normal pattern would be to let the user click it and then explain to them why they can't continue or what they need to do.

some of the problems with disabled buttons are:
1. they can look like they are clickable confusing the user
2. there's no feedback telling them what they are doing wrong if they try to click it

https://govuk-elements.herokuapp.com/buttons/#button-disabled
aria-disabled is missing here

Reply:
spoke with Pete about this and user testing sessions showed more confusion when not using disabled buttons.